widget/tabcontainer.go

```diff
@@ -46,9 +46,10 @@ const (
 type TabContainer struct {
 	BaseWidget
 
-	Items       []*TabItem
-	current     int
-	tabLocation TabLocation
+	Items         []*TabItem
+	OnTabSelected func(tab *TabItem)
```

Comment:
Rename to OnChanged to be consistent with Check, Radio, and Select.

I'm not sure whether this parameter should be the TabItem, or just the current index. Anyone have a strong opinion one way or the other?

Reply:
Is on OnChanged the best name? I know that it doesn’t provide as much consistency, but I think OnSelect might be a better name?

Reply:
I think TabItem is more useful - you can directly set different icon/title on selected tab. Index is rarely needed, and also you can get it in one call - so I don't see a problem.

Reply:
I agree TabItem - i think the rule of thumb is prefer the typed version when possible :)